ns-3_adapter/package.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<package>
+  <name>ns-3_adapter</name>
+  <version>0.0.0</version>
```

Comment:
Recommend version 0.0.1, and adding to the description that this is for use with CARMA XIL.